concrete-core/docs/user/introduction.md

```diff
@@ -0,0 +1,72 @@
+# Introduction
+
+Welcome to the `concrete-core` guide !
+
+This library contains a set of fast, low-level primitives which can be used to implement *Fully
+Homomorphically Encrypted* (FHE) programs. In a nutshell, fully homomorphic encryption makes it possible to
```

Comment:
Fully Homomorphically Encrypted -> Fully Homomorphically Encryption

Reply:
"In a nutshell, fully homomorphic encryption makes it" -> "In a nutshell, FHE makes it"